index.md

```diff
@@ -57,6 +57,7 @@ Along with bug fixes and general improvements, Anbox Cloud 1.13 comes with:
 | 2 | howto/update/upgrade-anbox | [Upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750)|
 | 1 | howto/manage/landing| [Manage Anbox Cloud](https://discourse.ubuntu.com/t/manage-anbox-cloud/24337) |
 | 2 | howto/manage/manage-appliance | [Manage the appliance](https://discourse.ubuntu.com/t/manage-the-anbox-cloud-appliance/26725) |
+| 2 | howto/manage/appliance-tls-setup | [Setup TLS for appliance](TBD) |
```

Comment:
```suggestion
| 2 | howto/manage/appliance-tls-setup | [Set up TLS for appliance](TBD) |
```

(It's "to set up" as a verb and "the setup" as a noun - same for other words like "to back up / the backup" or "to log in / the login" and so on.)